app/models/event-invoice.js

```diff
@@ -0,0 +1,39 @@
+import attr from 'ember-data/attr';
+import moment from 'moment';
+import ModelBase from 'open-event-frontend/models/base';
+import { belongsTo } from 'ember-data/relationships';
+
+const detectedTimezone = moment.tz.guess();
+
+export default ModelBase.extend({
+
+  /** attributes
+   *
+   */
+
+  status        : attr('string'),
+  identifier    : attr('string'),
+  paypalToken   : attr('string'),
+  transactionId : attr('string'),
+  brand         : attr('string'),
+  createdAt     : attr('moment', { defaultValue: () => moment.tz(detectedTimezone) }),
```

Comment:
@CosmicCoder96 Will the frontend create invoices or the server will take care of it?